Joke Machine/Controller/ViewController.swift

```diff
@@ -14,9 +14,7 @@ class ViewController: UIViewController {
     
     override func viewDidLoad() {
```

Comment:
yukarı yorum atamadığım için buraya yazıyorum IBOutletler private olarak tanımlanması encapsulation açısından uygun @IBOutlet private weak var blabla yazabilirsin.

Reply:
+

Reply:
++